action.yml

```diff
@@ -0,0 +1,57 @@
+# azure spirng cloud action
+name: 'AzureSpringCloud'
+description: 'Deploy applications to Azure Spring Cloud and manage deployments.'
+inputs: 
+  azure-subscription:
+    description: 'Select the Azure Resource Manager subscription for the deployment.'
+    required: true
+  action:
+    description: 'Action to be performed on Azure Spring Cloud.'
+    required: true
+    default: 'Deploy'
+  service-name:
+    description: 'Select the Azure Spring Cloud service to which to deploy.'
```

Comment:
Select the Azure Spring Cloud service to be deployed to.

Reply:
Just curious, is the description copied from the Azure DevOps pipeline tasks? If yes, we can leave them as it is to make the two consistent.

Reply:
Yes, the descriptions are copied from Azure DevOps pipeline tasks